File: Java Codewars/7kyu/Sum of numbers from 0 to N.java
/*
Description

We want to generate a function that computes the series starting from 0 and ending until the given number.

Example:
Input: > 6
Output: 0+1+2+3+4+5+6 = 21
Input: > -15
Output: -15<0
Input: > 0
Output: 0=0
*/

public class SequenceSum {

  public static String showSequence(int value) {
    if (value < 0) {
      return value + "<0";
    } else if (value == 0) {
      return "0=0";
    }
    int sum = 0;
    String result = "0";
    for (int i = 1; i <= value; i++) {
      result += "+" + i;
      sum += i;
    }
    return result += " = " + sum;
  }
  /*public static void main(String args[]){
    int param = Integer.ParseInt(args[0]);
    SequenceSum.showSequence(param);
  }*/
}

File: C# Codewars/7kyu/Sum of numbers from 0 to N.cs
/*
Description

We want to generate a function that computes the series starting from 0 and ending until the given number.

Example:
Input:
> 6
Output:
0+1+2+3+4+5+6 = 21
Input:
> -15
Output:
-15<0
Input:
> 0
Output:
0=0
*/

using System;

public class SequenceSum
{
    public static string ShowSequence(int n)
    {
        if (n < 0)
        {
            return n + "<0";
        }
        else if (n == 0)
        {
            return "0=0";
        }

        int sum = 0;
        string result = "0";

        for (int i = 1; i <= n; i++)
        {
            result += "+" + i;
            sum += i;
        }

        return result + " = " + sum;
    }
}